src/graphql/user/ExternalLogoutToken.graphql

```diff
@@ -0,0 +1,8 @@
+mutation ExternalLogoutToken($externalToken: String!) {
+  ExternalLogoutToken(externalToken: $externalToken) {
```

Comment:
Just a small note here because we usually use camel case for queries and mutations and here you start with capital letter which doesn't follow our style. Is there any specific reason for it or you can change it to `externalLogoutToken` instead of `ExternalLogoutToken`?

Reply:
Sorry mistake on my part I will change it